build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -387,6 +392,11 @@ antreaProxy:
   # Note that this option is experimental. If kube-proxy is removed, option kubeAPIServerOverride must be used to access
   # apiserver directly.
   proxyAll: {{ .proxyAll }}
+  # hostNetworkMode tells antrea-agent how to enforce netfilter rules required by `proxyAll` in the Node's host network.
```

Comment:
maybe here we don't need to repeat that this is specifically for `proxyAll` (we can keep that information only at the FeatureGate level for the config)